.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{kt,kts}]
+indent_size = 4
+max_line_length=off
+
+[*.dart]
+indent_brace_style = K&R
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false

```

Comment:
Why false?

Reply:
Sir i thought newline/whitespace will be preferred in .md files that's why is set `trim_trailing_whitespace = false`  .  but sir if you want i can set it to true .

Reply:
Look for best practices and do that.

Reply:
@aashutoshrathi okay sir i am looking into it.

Reply:
@aashutoshrathi i had a venture through some of the project i had modified editorconfig accordingly.